common/enablers/enableEpochsHandler_test.go

```diff
@@ -140,109 +141,111 @@ func TestNewEnableEpochsHandler_EpochConfirmed(t *testing.T) {
 		handler, _ := NewEnableEpochsHandler(cfg, &epochNotifier.EpochNotifierStub{})
 		require.False(t, check.IfNil(handler))
 
-		handler.EpochConfirmed(math.MaxUint32, 0)
+		currentEpoch := uint32(math.MaxUint32 - 1)
+		handler.EpochConfirmed(currentEpoch, 0)
```

Comment:
I think we can remove this call and change all the other calls to use currentEpoch instead of math.MaxUint32

Reply:
updated